src/Og.php

```diff
@@ -150,6 +150,8 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
       ->getStorage('og_membership')
       ->loadMultiple($results);
 
+    static::$entityGroupCache[$identifier]['node'] = [];
```

Comment:
We don't want to hard code, but also, doesn't the identifier have the $entity_type_id in anyway?